src-ts/tools/learn/welcome/progress-block/completed/Completed.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react'
+
+import styles from './Completed.module.scss'
+import { Button, LearnCertification, textFormatDateLocaleShortString } from '../../../../../lib'
```

Comment:
I don't think i noticed this before, but we shouldn't have anything specific to the learn tool, like the LearnCertification, in the lib. 

the only reason there are Work-specific components in the lib is bc they are used by the legacy /src self-service components. i debated having the work components in the work tool rather than the lib but decided on the shared lib temporarily. i prob should have just put them in the work tool.